### PR TITLE
Pin Cryptography.Cng

### PR DESCRIPTION
**Pull request by JunTaoLuo**
We decided to pin Cng as part of the discussion at https://github.com/dotnet/aspnetcore/pull/18364#discussion_r368208938.

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -38,9 +38,9 @@
       <Uri>https://github.com/dotnet/corefx</Uri>
       <Sha>4ac4c0367003fe3973a3648eb0715ddb0e3bbcea</Sha>
     </Dependency>
-    <Dependency Name="System.Security.Cryptography.Cng" Version="4.6.1" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
+    <Dependency Name="System.Security.Cryptography.Cng" Version="4.6.0" Pinned="true">
```
Comment (JunTaoLuo) [inline eng/Version.Details.xml]: Does this need to be moved as per: https://github.com/dotnet/aspnetcore/pull/18364#discussion_r369765860

Comment (dougbu) [reply eng/Version.Details.xml]: @mmitche are you implying by your approval that we do **not** need to move pinned dependencies below non-pinned ones from the same repo anymore?

Comment (wtgodbe) [reply eng/Version.Details.xml]: I believe @mmitche has pushed a fix that renders the ordering unnecessary here - can you confirm Matt?

Comment (wtgodbe) [reply eng/Version.Details.xml]: He confirmed in the AspNetCore PR, so this should be good to go

**Pull request merged**